geode-junit/src/main/java/org/apache/geode/test/version/TestVersion.java

```diff
@@ -17,18 +17,24 @@
 import java.io.Serializable;
 import java.util.Objects;
 
+@SuppressWarnings("serial")
 public class TestVersion implements Comparable<TestVersion>, Serializable {
-  public static final TestVersion CURRENT_VERSION = new TestVersion(VersionManager.CURRENT_VERSION);
+
+  static final TestVersion CURRENT_VERSION = new TestVersion(VersionManager.CURRENT_VERSION);
 
   private final int major;
   private final int minor;
   private final int release;
 
+  public static TestVersion current() {
+    return CURRENT_VERSION;
+  }
+
```

Comment:
It looks like this method was added to provide a little more encapsulation than exposing `CURRENT_VERSION` did. Only `VmConfiguration` continues to reference `CURRENT_VERSION`: I recommend making that class go through the method so `CURRENT_VERSION` can become `privsate`.

Reply:
Thanks! That was the intention but it looks like changing VmConfiguration was skipped somehow.

Reply:
Changed CURRENT_VERSION to private and VmConfiguration now uses current().